examples/example1.xml

```diff
@@ -34,15 +34,16 @@
 
     <ComponentType name="iaf2" extends="iaf1">
         <Fixed parameter="threshold" value="-45mV" />
+        <Exposure name="v" dimension="voltage" />
```

Comment:
@sanjayankur31 I can see why this would be useful for testing, but it might not be good to do this with the main examples in lems, simply because the exposure here is not linked to anything (like a derived variable) and so it's probably technically invalid. Maybe add a new example for testing these (with just iaf like components, and dummy values for v). Also, good to copy updated versions of these examples to: https://github.com/LEMS/LEMS/tree/master/examples

Reply:
Makes sense. Update in new commits. 

I'll file a different issue to discuss the migration of examples.